pkg/unpack/unpacker.go

```diff
@@ -409,14 +410,14 @@ func (u *Unpacker) unpack(
 	}
 
 	for i, desc := range layers {
-		_, layerSpan := tracing.StartSpan(ctx, "pkg.unpack.unpacker.unpackLayer")
+		_, layerSpan := tracing.StartSpan(ctx, tracing.SpanName(unpackSpanPrefix, "unpackLayer"))
```

Comment:
Too many "spans", possibly:

```go

layerSpan.SetAttributes(
    tracing.Attribute("layer.media.type", desc.MediaType),
    tracing.Attribute("layer.media.size", desc.Size),
    tracing.Attribute("layer.media.digest", desc.Digest.String()),
)
```

Reply:
What I mean here is whether we could rename `tracing.SpanAttribute` to `tracing.Attribute`

Reply:
Ahh I see. I have changed `SpanAttribute` to `Attribute`.